packages/webapp/src/containers/Map/useSelectionHandler.js

```diff
@@ -68,10 +68,14 @@ const useSelectionHandler = () => {
             });
             dispatch(canShowSelection(true));
             dispatch(locations(locationArray));
-          } else {
+          } else if (overlappedLocations.point[0].type === 'sensor') {
```

Comment:
I should have used a constant on line 62. Can we have line 62 and 71 use constants for 'sensor' in this PR?